<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Referral</class>
 <widget class="QWidget" name="Referral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_title">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>361</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Insert referral data</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_id">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>30</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Generated ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_patient_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Patient data</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_patient_data">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>501</width>
     <height>71</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_dest">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Referral destination specialization</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="text_edit_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>501</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_description">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Purpose of referral</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_add">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>510</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="push_button_return">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>510</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Return</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_box_codes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_code">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>181</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Code and issue</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_dest_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Justification</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_edit_dest_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>450</y>
     <width>501</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Comment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
